### ISSUE TITLE
Crash on 1.20.6 when putting non-shield in offhand

### ISSUE THREAD

**Issue by srnyx** [MightyKnight/ShieldDisruptor#3]
When I put a shield in my offhand, it doesn't crash, but when I put anything else in my offhand, my entire game crashes.

Crash log: https://paste.srnyx.com/ihenozorig.txt
Mod version: 1.8.0
Cloth Config API version: 14.0.126+fabric

Let me know if you need any other info

**Comment by MightyKnight** [comment]
Oh this is aggrevating. For some reason it doesn't crash in my dev environment, which is why I didn't catch it during testing.
Thank you very much for reporting.

 I know exactly why this is happening, should be able to fix this today.




<details>
<summary>
<b>Open to read important crash info</b> (to preserve it, since hastebin deletes logs)
</summary>

```
java.lang.NoSuchMethodError: 'net.minecraft.class_2960 net.minecraft.class_2960.method_60655(java.lang.String, java.lang.String)'
	at net.minecraft.class_759.handler$ekb000$shield-disruptor$hideShield(class_759.java:3284)
	at net.minecraft.class_759.method_3233(class_759.java)
	at net.minecraft.class_989.method_4192(class_989.java:64)
	at net.minecraft.class_5697.method_4192(class_5697.java:35)
	at net.minecraft.class_989.method_17162(class_989.java:43)
	at net.minecraft.class_989.method_4199(class_989.java:15)
	at net.minecraft.class_922.method_4054(class_922.java:149)
	at net.minecraft.class_1007.method_4215(class_1007.java:68)
	at net.minecraft.class_1007.method_3936(class_1007.java:45)
	at net.minecraft.class_898.method_3954(class_898.java:143)
	at net.minecraft.class_490.method_29977(class_490.java:156)
	at com.mojang.blaze3d.systems.RenderSystem.runAsFancy(RenderSystem.java:1190)
	at net.minecraft.class_490.method_48472(class_490.java:156)
	at net.minecraft.class_490.method_2486(class_490.java:130)
	at net.minecraft.class_490.method_2389(class_490.java:99)
	at net.minecraft.class_465.method_25420(class_465.java:172)
	at net.minecraft.class_437.method_25394(class_437.java:129)
	at net.minecraft.class_465.method_25394(class_465.java:100)
	at net.minecraft.class_485.method_25394(class_485.java:32)
	at net.minecraft.class_490.method_25394(class_490.java:81)
	at net.minecraft.class_437.method_47413(class_437.java:120)
	at net.minecraft.class_757.mixinextras$bridge$method_47413$298(class_757.java)
	at net.minecraft.class_757.wrapOperation$bog000$fabric-screen-api-v1$onRenderScreen(class_757.java:5601)
	at net.minecraft.class_757.method_3192(class_757.java:922)
	at net.minecraft.class_310.method_1523(class_310.java:1320)
	at net.minecraft.class_310.method_1514(class_310.java:888)
	at net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
```

</details>

**Comment by MightyKnight** [comment]
This is now fixed in the 1.8.1 release: [Modrinth](https://modrinth.com/mod/shielddisruptor/version/ncfNJCZ2), [Github](https://github.com/MightyKnight/ShieldDisruptor/releases/tag/1.8.1)

Please test it on your side, and thank you again.

**Comment by srnyx** [comment]
Yep, works, thank you!
